src/sections/PastSpeakers.vue

```diff
@@ -1,16 +1,129 @@
 PastSponsors.vue<template>
   <Section id='past-speakers' as='section'>
-    owo
+    <div>
+      <Typography class='past-speakers__heading' type='heading2' color='light-yellow' transform='uppercase' as='h2'>
+        Past Speakers
+       <img class='past-speakers__bubble' src="../assets/past-speakers/megaphone-bubble.svg">
+      </Typography>
+    </div>
+    <div class="past-speakers__speakers">
+      <div class="past-speakers__card">
+        <Placeholder class='past-speakers__placeholder past-speakers__placeholder--Jonathan' width='365px' height='640px' color='light-teal'/>
+        <div class="past-speakers__label">
+          <Typography type='heading3' color='black' as='h3'>
+            Jonathan Javier
+          </Typography>
+          <Typography type='heading4' color='black' as='h4'>
+            CEO/Founder @Wonsulting
+          </Typography>
+        </div>
+      </div>
+      <div class="past-speakers__card">
+        <Placeholder class='past-speakers__placeholder past-speakers__placeholder--Cathy' width='365px' height='640px' color='light-teal'/>
+        <div class="past-speakers__label">
+          <Typography type='heading3' color='black' as='h3'>
+            Cathy Tie
+          </Typography>
+          <Typography type='heading4' color='black' as='h4'>
+            Serial Entreprenuer & VC
+          </Typography>
+        </div>
+      </div>
+      <div class="past-speakers__card">
+        <Placeholder class='past-speakers__placeholder past-speakers__placeholder--Pei' width='365px' height='640px' color='light-teal'/>
+        <div class="past-speakers__label">
+          <Typography type='heading3' color='black' as='h3'>
+            Pei Li
+          </Typography>
+          <Typography type='heading4' color='black' as='h4'>
+            Founder @Hack the 6ix
+          </Typography>
+        </div>
+      </div>
```

Comment:
You can consider having an array of objects represent this info, and using `v-for` instead